persist-mongodb/src/main/kotlin/io/saagie/updatarium/persist/model/MongoDbChangeSet.kt

```diff
@@ -18,21 +18,32 @@
 package io.saagie.updatarium.persist.model
 
 import io.saagie.updatarium.model.ChangeSet
+import io.saagie.updatarium.model.ChangeSetParameters
 import io.saagie.updatarium.model.Status
 import java.time.Instant
 
+data class MongoDbChangeSetParameters(
```

Comment:
I think that we does not need this data we can use `ChangeSetParameters` instead

Reply:
yes, I decided to comply with the existing choice but it is indeed possible to use ChangeSetParameters directly. I'll modify that :+1:

Reply:
done